python/ray/experimental/serve/api.py

```diff
@@ -0,0 +1,178 @@
+import inspect
+
+import numpy as np
+
+import ray
+from ray.experimental.serve.task_runner import RayServeMixin, TaskRunnerActor
+from ray.experimental.serve.utils import pformat_color_json, logger
+from ray.experimental.serve.global_state import GlobalState
+
+global_state = GlobalState()
+
+
+def init(blocking=False, object_store_memory=int(1e8)):
+    """Initialize a ray serve cluster.
+
+    Args:
+        blocking (bool): If true, the function will wait for HTTP server to be
+            healthy before returns
+        object_store_memory (int): Allocated shared memory size. The default
+            is 100MB. Unit in bytes.
+    """
+    if not ray.is_initialized():
```

Comment:
Not sure this is the right behavior. We should probably require that `ray.init()` be called before this - what if they want to pass different params?

Reply:
User can call `ray.init()` before `serve.init` if they want. `serve` will just use the current ray cluster. They can pass different params. I will document this.